datahub-web-react/src/app/search/filters/utils.tsx

```diff
@@ -190,14 +191,26 @@ export function getFilterIconAndLabel(
         icon = <FolderFilled size={size} color="black" />;
         label = getLastBrowseEntryFromFilterValue(filterValue);
     } else if (filterEntity) {
-        const { icon: newIcon, label: newLabel } = getFilterWithEntityIconAndLabel(
-            filterValue,
-            entityRegistry,
-            filterEntity,
-            size,
-        );
-        icon = newIcon;
-        label = newLabel;
+        // Scenario where the filter entity exists and filterField is container
+        if (filterField === CONTAINER_FILTER_NAME) {
```

Comment:
please elevate this condition to the same level as 

else if (filterField === PLATFORM_FILTER_NAME) 

to keep things consistent (instead of nesting it down here)

Reply:
Updated the condition